gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -501,20 +504,16 @@ describe('loadSupergraphSdlFromStorage', () => {
       );
     });
 
-    it('throws on 400 status response and successfully submits an out of band error', async () => {
-      cleanUp = mockedEnv({
-        APOLLO_OUT_OF_BAND_REPORTER_ENDPOINT: mockOutOfBandReporterUrl,
-      });
-
+    it('throws on 400 status response and does not submit an out of band error', async () => {
```

Comment:
Throwing on 400 and not reporting does make sense because I don't foresee many scenarios where we would care that someone sent a bad request. But for now we can report it and look at the traffic we collect to make further decisions.

Reply:
for now let's just keep the current behaviour of not reporting 400s per slack convo @trevor-scheer